lectures/5-logic/script.js

```diff
@@ -2,12 +2,12 @@ const FIRST_VALUE_ELEMENT = document.getElementById('firstValue');
 const SECOND_VALUE_ELEMENT = document.getElementById('secondValue');
 const OPERATION_ELEMENT = document.getElementById('operation');
 const CALCULATE_BUTTON_ELEMENT = document.getElementById('calculate');
-const RESULT_ELEMENT = document.getElementById('result');
+const IMAGE_ELEMENT = document.getElementById('result');
```

Comment:
Here also. 
Would be very useful to check your pr files before sending the pr to review
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/047b3ae7-e373-4999-ba3f-3b77607710e0)

Reply:
@vladyslav-yermolin-moc fixed

Reply:
@vladyslav-yermolin-moc agreed with you. accepted